pulp_smash/constants.py

```diff
@@ -343,11 +344,74 @@
 .. _Repository APIs:
     https://docs.pulpproject.org/en/latest/dev-guide/integration/rest-api/repo/index.html
 """
-
-RPM = 'bear-4.1-1.noarch.rpm'
+RPM_DATA = MappingProxyType({
+    'name': 'bear',
+    'epoch': '0',
+    'version': '4.1',
+    'release': '1',
+    'arch': 'noarch',
+    'vendor': None,
+    'metadata': {
+        'release': '1',
+        'license': 'GPLv2',
+        'description': 'A dummy package of bear',
+        'files': {'dir': [], 'file': ['/tmp/bear.txt']},
+        'group': 'Internet/Applications',
+        'size': '42',
+        'sourcerpm': 'bear-4.1-1.src.rpm',
+        'summary': 'A dummy package of bear',
+    },
+})
+"""Metadata for an RPM with an associated erratum.
+
+The metadata tags that may be present in an RPM may be printed with:
+
+.. code-block:: sh
+
+    rpm --querytags
+
+Metadata for an RPM can be printed with a command like the following:
+
+.. code-block:: sh
+
+    for tag in name epoch version release arch vendor; do
+        echo "$(rpm -qp bear-4.1-1.noarch.rpm --qf "%{$tag}")"
+    done
+
+"""
+RPM = '{}-{}{}-{}.{}.rpm'.format(
+    RPM_DATA['name'],
+    RPM_DATA['epoch'] + '!' if RPM_DATA['epoch'] != '0' else '',
+    RPM_DATA['version'],
+    RPM_DATA['release'],
+    RPM_DATA['arch'],)
```

Comment:
Thanks for explaining the motivation. 

One thing I'm having trouble with in the vein of formatting is getting hanging closing braces done right. There are cases, especially when they are nested, that I can't seem to make both linters happy so I have resorted to not putting the closing brace on a new line, as in the snippet above. Any advice?